react-function-components/src/index.jsx

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+
+function CustomButton() {
+  return (
+  <button>Click me</button>
+  );
+}
+
+ReactDOM.render(CustomButton(), document.getElementById('root'));
```

Comment:
When writing React code, we generally never call our own components. Instead create a react element.

```jsx
ReactDOM.render(<CustomButton />, document.getElementById('root'));

```